src/components/report-technical-issue/template.njk

```diff
@@ -4,4 +4,4 @@
   class="govuk-link {{ params.classes }}"
   target="_blank"
   href="{{ params.baseUrl }}/contact/problem_reports_nonjs?newTab=true{% if params.serviceCode %}&amp;service={{ params.serviceCode | urlencode }}{% endif %}{% if params.referrerUrl %}&amp;referrerUrl={{ params.referrerUrl | urlencode }}{% endif %}"
->{% if params.language == 'cy' %}Help gyda'r dudalen hon (yn agor ffenestr neu dab newydd){% else %}Get help with this page (opens in new tab){% endif %}</a>
+>{% if params.language == 'cy' %}A yw’r dudalen hon yn gweithio’n iawn? (yn agor mewn tab newydd){% else %}Is this page not working properly? (opens in a new tab){% endif %}</a>
```

Comment:
I'm not exactly sure of the reasons but gov.uk design system says wording should be "opens in new tab" as opposed to "opens in a new tab". We're archiving the pattern in our design system soon.